src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");

```

Comment:
The message 'Truck started its work.' is not very informative. Consider specifying what work the Truck is doing to make the message more informative, adhering to the checklist point about writing informative messages in methods.